sdk/python/generator/Generator.py

```diff
@@ -113,73 +94,59 @@ def bind_size_fields(struct_type):
 	for field in struct_type.layout:
 		if field.type_instance.is_array and isinstance(field.get_type().size, str):
 			size_field_name = field.get_type().size
-			# 'unfix' name when searching for associated 'size' field:
-			#  * fields are added using 'original' name,
-			#  * but 'size' property of given array type is already fixed
-			#
-			# (currently this only happens for metadata value)
-			if 'size_' == size_field_name:
-				size_field_name = 'size'
-
```

Comment:
where did this go? :D

Reply:
benefit of moving to printers stage, is that we don't need these fixups anymore.